rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/MUTEExamplesTest.kt

```diff
@@ -20,13 +20,8 @@ class MUTEExamplesTest {
     }
 
     @Test @Category(PerformanceTest::class)
-    fun executeMUTE10_04_A_perf_strings() {
-        assertEquals(listOf("001.1_d01.1_A01.1_c01.1_B01.1_b01.1_C01.1_901.1_101."), outputOf("performance/MUTE10_04_A"))
-    }
-
-    @Test @Category(PerformanceTest::class)
-    fun executeMUTE11_02() {
```

Comment:
Why this test has been removed?

Reply:
MUTE11_02 is not a performance test. Is a program to test integration in the Smeup Gateway.

Reply:
Marco, that's true, but maybe we could keep it as a performance test too, don't you think?

Reply:
But it's a program that was created to run within the gateway as a folder listener. I don't think it can be useful as a performance test. It can't run correctly.

Reply:
OK, good point